src/viking.js

```diff
@@ -9,18 +9,46 @@ class Soldier {
     return this.strength;
   }
 
-  receiveDamage(thedamage) {
-    this.health -= thedamage;
+  receiveDamage(attackDamage) {
+    this.health -= attackDamage;
   }
 }
 
 // Viking
-class Viking {
+
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  receiveDamage(attackDamage) {
+    super.receiveDamage(attackDamage);
+    if (this.health > 0) {
+      return `${this.name} has received ${attackDamage} points of damage`;
+    } else {
+      return `${this.name} has died in act of combat`;
+    }
+  }
+
+  battleCry() {
+    return "Odin Owns You All!";
+  }
```

Comment:
Everton, aqui na linha 26, vc não precisa chamar o método dentro do método, pode só deixar mesmo assim: 
this.health -= attackDamage